.gitignore

```diff
@@ -6,3 +6,10 @@ package-lock.json
 storybook-static
 yarn-error.log
 **/coverage
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

Comment:
what is the meaning of some of these subdirectories like patches/releases/versions? They almost seem related to the way the monorepo itself is publishing packages to registries?

Reply:
I followed the advice of this article https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

They have more details for each line there!